v3/docs/01-getting-started/a-overview/index.mdx

```diff
@@ -1,66 +1,61 @@
 ---
 id: 001
-title: Welcome
+
+title: Introduction to Substrate
 slug: /v3/getting-started/overview
 version: '3.0'
 section: docs
 category: getting started
 keywords: []
 ---
+# Overview
+
+This section provides an overview of Substrate – structure, working, and usage. 
+
+## Prerequisites
+
+To get the most out of Substrate, you should have a good knowledge of computer science and basic blockchain concepts in addition to terminologies like header, block, client, hash, transaction, and signature. Substrate is built on the Rust programming language, which uses novel design patterns to enable development of code that is safe and fast.
+
+>**Note:** Although you don't need to know Rust to get started with Substrate, a good understanding of Rust will enable you to become a better Substrate developer. To to build your Rust development skills, see [Get Started with Rust](https://www.rust-lang.org/learn) provided by the Rust community. 
+
+## What is Substrate?
+
+Substrate is an extensible, modular, open-source framework with an approach to build (customized) blockchains. It defines a rich set of primitives that allows developers to make use of powerful, familiar programming idioms. Substrate provides developers with all the core components that are required to build a distributed blockchain network. Before we dive into Substrate, refer to the [Architecture](https://docs.substrate.io/v3/getting-started/architecture/) section for more details on architecture (or concepts) of Substrate blockchain development.
+
+### Structure
+
+Blockchains are decentralized and there are many nodes connected on a peer-to-peer network – there is no central authority. Each node is running a blockchain program and stores a copy of the chain. When there is a set of transactions, a node can produce a new block and connect it to its chain. All nodes must accept the new block by solving a mathematical problem, which takes much computational power. 
+
```

Comment:
It might be better to have a more general description of blockchains as decentralized networks before you describe Substrate as a framework for building customized blockchains.
Also, the part about solving a mathematical problem is specifically "proof of work" consensus, so it would be better to leave that out.